src/pip/_internal/models/direct_url.py

```diff
@@ -71,21 +71,20 @@ class VcsInfo:
 
     def __init__(
         self,
-        vcs,  # type: str
-        commit_id,  # type: str
-        requested_revision=None,  # type: Optional[str]
-        resolved_revision=None,  # type: Optional[str]
-        resolved_revision_type=None,  # type: Optional[str]
-    ):
+        vcs: str,
+        commit_id: str,
+        requested_revision: Optional[str] = None,
+        resolved_revision: Optional[str] = None,
+        resolved_revision_type: Optional[str] = None,
+    ) -> None:
```

Comment:
The original annotation didn't say anything about a return value, but I added it as `-> None`.

Reply:
The other 3 reviews after this one are about the same, but in another places of `direct_url.py`.